client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -63,9 +63,10 @@ export class HubConnection {
                     break;
                 case MessageType.Result:
                 case MessageType.Completion:
+                case MessageType.StreamCompletion:
                     let callback = this.callbacks.get(message.invocationId);
                     if (callback != null) {
-                        if (message.type == MessageType.Completion) {
+                        if (message.type == MessageType.Completion || message.type == MessageType.StreamCompletion) {
```

Comment:
`===` instead of `==`?

Reply:
Actually, this is inside switch so it does not matter.

Reply:
Usage of `===` is generally [always preferred](https://palantir.github.io/tslint/rules/triple-equals/) no matter where it occurs (even if it would be "safe" to use `==`). It's a code style and clarity thing rather than a correctness thing. We should consider running a linter like [TSLint](https://palantir.github.io/tslint/) on our code